### ISSUE TITLE
[Bug Report] v-textarea inside v-expansion-panel does not auto-grow

### ISSUE THREAD

**Issue by peterjungx** [vuetifyjs/vuetify#7123]
### Versions and Environment
**Vuetify:** 1.5.14
**Vue:** 2.6.10
**Browsers:** Chrome 73.0.3683.103
**OS:** Mac OS 10.14.4

### Steps to reproduce
Place a v-textarea with auto-grow, rows="1" and with a multiline-text inside an v-expansion-panel.

### Expected Behavior
The view should grow to the content size automatically.

### Actual Behavior
When revealing the expansion panel content, view stays at 1 rows display.

### Reproduction Link
<a href="https://codepen.io/peterjung/pen/pBXmvg" target="_blank">https://codepen.io/peterjung/pen/pBXmvg</a>

### Other comments
As soon as the text in the textarea is edited, the size is recalculated and the display is correct.

**Comment by KaelWD** [comment]
See #5849

tl;dr use `lazy`

**Comment by peterjungx** [comment]
Fantastic, thanks!
